apps/pie-docs/src/assets/styles/components/_list.scss

```diff
@@ -52,6 +52,29 @@
     }
 }
 
+.c-list--bullet {
+    margin: 0;
+
+    .c-list-item {
+        @include f.font-size('body-l');
+        display: flex;
+        align-items: flex-start;
+        justify-content: flex-start;
+        margin-bottom: f.spacing(b);
+
+        &:before {
+            content: '';
+            width: 6px;
+            height: 6px;
+            background: f.$color-container-inverse;
+            border-radius: f.$radius-rounded-e;
+            margin-right: f.spacing(b);
+            margin-top: f.spacing(b);
```

Comment:
is it worth changing the margin-top/right/bottom/lefts to `margin-block-start/end` and `margin-inline-start/end`

Reply:
done ☑️ 

I think we don't use them in the whole codebase of the pie docs btw, do you think we should create a ticket to address that at some point?

Reply:
Yep, I can add that to the to do list 😄